jail-docs.yml

```diff
@@ -0,0 +1,51 @@
+---
+- name: configure docs
+  hosts: docs
+  gather_facts: no
+  connection: local
+  sudo: yes
+  vars:
+    env_name: doctools
+    web_host_name: "docs.buildbot.net"
+    bbdocs_dir: "bbdocs"
+
+  roles:
+  - base
+  - role: user
+    user_id: "{{ worker_account }}"
+    user_name: Buildbot Worker Account
+  - role: virtualenv
+    venv_user: "{{ worker_account }}"
+    venv_home_dir: "{{ getent_passwd[worker_account].4 }}"
+    venv_name: "{{ env_name }}"
+    venv_python_packages:
+    - 'sphinx==1.2.2'
+    - 'Pygments==2.0.1'
+  - role: simple-buildout
+    repo_url: git://github.com/buildbot/bbdocs.git
+    repo_branch: master
+    target_user: "{{ worker_account }}"
+    target_dir: "{{ getent_passwd[worker_account].4 }}/{{ bbdocs_dir }}"
+  - role: simple-buildout
+    repo_url: git://github.com/buildbot/buildbot.git
+    repo_branch: master
+    target_user: "{{ worker_account }}"
+    target_dir: "{{ getent_passwd[worker_account].4 }}/buildbot"
+    target_commands:
+    # `command` module does not allow to set env variables, nor expands them,
+    # so let's use make's variables.
+    - "make -C master/docs PATH:={{ getent_passwd[worker_account].4 }}/{{ env_name }}/bin:$(PATH) VERSION=latest"
+    - "find master/docs/_build/html -name '*.html' -exec python2 {{ getent_passwd[worker_account].4 }}/{{ bbdocs_dir }}/add-tracking.py {} ;"
+    - "ln -sf {{ getent_passwd[worker_account].4 }}/buildbot/master/docs/_build/html {{ getent_passwd[worker_account].4 }}/{{ bbdocs_dir }}/docs/latest"
```

Comment:
This is nice - we can eliminate the uploads from the 'docs' builder in metabuildbot.